codegen/files/rest/server.go

```diff
@@ -1,11 +1,587 @@
 package rest
 
 import (
+	"fmt"
+	"text/template"
+
 	"goa.design/goa.v2/codegen"
+	"goa.design/goa.v2/codegen/files"
+	"goa.design/goa.v2/design"
 	"goa.design/goa.v2/design/rest"
+	"net/http"
+	"sort"
+	"strings"
+)
+
+//todo: in encode headers and mappings
+
+const serverStructT = `{{ printf "%s lists the %s service endpoint HTTP handlers." .HandlersStruct .ServiceName | comment }}
+type {{ .HandlersStruct }} struct {
+	{{- range .ActionData }}
+	{{ .EndpointName }} http.Handler
+	{{- end }}
+}
+`
+
+const serverConstructorT = `{{ printf "%s instantiates HTTP handlers for all the %s service endpoints." .Constructor .ServiceName | comment }}
+func {{ .Constructor }}(
+	e *endpoints.{{ .ServiceName }},
```

Comment:
OK then the version used in the comment is wrong :) Might be good to be consistent and follow the `X` vs `VarX` convention for the non-goified and goified versions.